src/pages/signup/StepThree.jsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
+import { TextField } from '@mui/material';
+import { useState } from 'react';
+
+export default function StepThree(props) {
+
+    const [value, setValue] = useState("");
+
+    const handleChange = (e) => {
+        setValue(e.target.value);
+    }
```

Comment:
The `handleChange` function is being shared with the rest of the inputs on this page so when one changes it will overwrite any previous values. You'll probably want to handle the values of the inputs like you did in `StepOne.jsx`